src/components/calculator/Calculator.js

```diff
@@ -1,13 +1,14 @@
-import React, { useState } from 'react';
-import calculate from '../../logic/calculate';
-import './calculator.css';
+/*eslint-disable */
```

Comment:
- [ ] @JakariaJishan great job implementing the changes requested however you have disable linter checks while implementing the changes which is bad practise, kindly fix the linter errors instead of disabling the check.